### PR TITLE
Add MySQL 8.0.26 Support

### PR DESCRIPTION
**Pull request by d-kuro**
refs: https://github.com/cybozu-go/moco/issues/285

### CODE REVIEW

File: DEVELOP.md [hunk 1]
```diff
@@ -54,10 +54,10 @@ Edit the following lines in `Dockerfile`:
 
 ```
 # The tag should be the latest one
-FROM quay.io/cybozu/mysql:8.0.25.1 as mysql
+FROM quay.io/cybozu/mysql:8.0.26.2 as mysql
```
Comment (d-kuro) [inline DEVELOP.md:57]: refs: https://quay.io/repository/cybozu/mysql?tab=tags

File: Dockerfile [hunk 2]
```diff
@@ -18,13 +18,13 @@ USER 10000:10000
 ENTRYPOINT ["/moco-controller"]
 
 # For MySQL binaries
-FROM quay.io/cybozu/mysql:8.0.25.1 as mysql
+FROM quay.io/cybozu/mysql:8.0.26.1 as mysql
```
Comment (ymmt2005) [inline Dockerfile:21]:
```suggestion
FROM quay.io/cybozu/mysql:8.0.26.2 as mysql
```
?

**Review by ymmt2005 — CHANGES_REQUESTED** [review]
Please add support for 8.0.26 in https://github.com/cybozu-go/moco-agent as well.

File: Dockerfile (hunk 2 above)
Comment (d-kuro) [reply Dockerfile:21]: Thank you followup 🙏

> Please add support for 8.0.26 in https://github.com/cybozu-go/moco-agent as well.

I will do it.

**Review by ymmt2005 — APPROVED** [review]
LGTM

**Comment by d-kuro** [comment]
Created: https://github.com/cybozu-go/moco-agent/pull/61

**Pull request merged**